ugs-core/src/com/willwinder/universalgcodesender/GrblController.java

```diff
@@ -261,6 +261,7 @@ else if (GrblUtils.isGrblStatusString(response)) {
                 verbose = true;
                 
                 this.handleStatusString(response);
+                this.checkStreamFinished();
```

Comment:
We now need to check if a file stream has finished when the controller state changes.